pod/perlvar.pod

```diff
@@ -779,6 +779,67 @@ and use an C<END{}> or CORE::GLOBAL::die override instead.
 See L<perlfunc/die>, L<perlfunc/warn>, L<perlfunc/eval>, and
 L<warnings> for additional information.
 
+=item %{^HOOK}
+X<%{^HOOK}>
+
+This hash contains coderefs which are called when various perl keywords
+which are hard or impossible to wrap are called. The keys of this hash
+are named after the keyword that is being hooked, followed by two
+underbars and then a phase term; either "before" or "after".
+
+It is an error to attempt to a value into this hash under a key which is
+not known to perl, and it is an error to store anything other than a
+CODEREF or undef into the hash. If you wish to use an object to implement
+a hook you can use currying to embed the object into the subroutine.
```

Comment:
"attempt to XXX a value into" - missing word, presumably "store"

"It is an error" - with what consequences?

Perhaps instead:
```
Perl will throw an error if you attempt to modify a key which is not
known to perl, or to store anything other than a CODEREF or undef
into the hash.
```

Reply:
Also, we seem to mostly spell out "code refererence" rather than "coderef" or "CODEREF" in prose in pod/

Reply:
Fixed, and i used your suggested wording with a few minor changes.